src/Arcus.Scripting.ApiManagement/Scripts/Import-AzApiManagementOperationPolicy.ps1

```diff
@@ -0,0 +1,12 @@
+param(
+    [string][parameter(Mandatory = $true)] $ResourceGroup,
+    [string][parameter(Mandatory = $true)] $ServiceName,
+    [string][parameter(Mandatory = $true)] $ApiId,
+    [string][parameter(Mandatory = $true)] $OperationId,
+    [string][parameter(Mandatory = $true)] $PolicyFilePath
+)
+
+$apimContext = New-AzApiManagementContext -ResourceGroupName $ResourceGroup -ServiceName $ServiceName
+
+Write-Host "Updating policy of the operation '$OperationId' in API '$ApiId'"
+Set-AzApiManagementPolicy -Context $apimContext -ApiId $ApiId -OperationId $OperationId -PolicyFilePath $PolicyFilePath
```

Comment:
Let's provide an update if the policy was successfully imported, such as 
`Write-Host 'Successfully updated the operation policy.'`